### PR TITLE
HW2: handled conflicts

### PR DESCRIPTION
**Pull request by Debi-Ejeta**
resolved conflicts

### CODE REVIEW

File: students/dme2136.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ UNI: dme2136
...
 Github: [link](https://github.com/Debi-Ejeta)
 
-## Project Proposal
+GitHub README: [link](https://github.com/Debi-Ejeta/Debi-Ejeta/blob/main/README.md)
 
-I am not sure about my project yet. I am thinking about something that has to do with image processing library.
 
-## Project Contributions
+## Project Proposal
+- [soccer-api-wrapper](./projects/python/soccer-api-wrapper.md)
```
Comment (timkpaine) [inline students/dme2136.md:13]: update link path and re-squash 😈

File: students/dme2136.md [hunk 2]
```diff
@@ -6,10 +6,11 @@ UNI: dme2136
...
 
-I am thinking of contributing to Flask and Numpy open source projects.
+## Project Contributions
+- [geopandas](./projects/python/geopandas.md)
```
Comment (timkpaine) [inline students/dme2136.md:16]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/dme2136.md (hunk 1 above)
Comment (Debi-Ejeta) [reply students/dme2136.md]: done!

File: students/dme2136.md (hunk 2 above)
Comment (Debi-Ejeta) [reply students/dme2136.md]: done!

**Pull request merged**